src/Transports.AspNetCore/WebSockets/WebSocketConnection.cs

```diff
@@ -123,9 +123,11 @@ public virtual async Task ExecuteAsync(IOperationMessageProcessor operationMessa
                         // queue the closure
                         _ = CloseAsync();
                         // wait until the close has been sent
-                        await Task.WhenAny(
-                            _outputClosed.Task,
-                            Task.Delay(_closeTimeout, RequestAborted));
+                        try
+                        {
+                            await _outputClosed.Task.WaitAsync(_closeTimeout, RequestAborted);
+                        }
+                        catch (TimeoutException) { }
```

Comment:
🤔 You catch only `TimeoutException` here since itis the only new one exception appeared but one more thing has changed. Old code throws `TaskCanceledException`, new code throws `OperationCanceledException` (base). Is it OK to consumers with `(catch TaskCanceledException)`? Should they use `(catch OperationCanceledException)` all the time?

Reply:
> Should they use `(catch OperationCanceledException)` all the time?

Absolutely.  While it may be that a task throws a TCE, methods like `CancellationToken.ThrowIfCancellation` throw a OCE.  So in my opinion, users should never `catch (TaskCanceledException)` and always catch the OCE.